gef.py

```diff
@@ -10647,6 +10684,22 @@ def invoke(self, args, from_tty):
         gdb.execute("gef help")
         return
 
+    def add_context_pane(self, pane_name, display_pane_function, pane_title_function):
+        """Add a new context pane to ContextCommand"""
+        for cmd, class_name, class_obj in self.loaded_commands:
+            if isinstance(class_obj, ContextCommand):
+                context_obj = class_obj
+                break
+
+        # assure users can toggle the new context
+        corrected_settings_name = pane_name.replace(" ", "_")
+        layout_settings = context_obj.get_setting("layout")
+        context_obj.update_setting("layout", "{} {}".format(layout_settings, corrected_settings_name))
+
+        # overload the printing of pane title
+        context_obj.layout_mapping[corrected_settings_name] = (display_pane_function, pane_title_function)
+        return True
```

Comment:
Question above about the return value. If you nix it there, just plain `return` here.